languages/javalang/builder-renderer/generated-code/src/test/java/com/commercetools/utils/CommercetoolsTestUtils.java

```diff
@@ -28,4 +32,30 @@ public static LocalizedString randomLocalizedString() {
     public static String getProjectKey() {
         return System.getenv("JVM_SDK_IT_PROJECT_KEY");
     }
+
+    public static String getClientId() {
+        return System.getenv("JVM_SDK_IT_CLIENT_ID");
```

Comment:
Maybe we can change the names of these environment variables to e.g. CTP_CLIENT_ID so that users of our sdk can just copy/paste environment variables that they get from Merchant Center ?

Reply:
sounds good to me, changed it to the prefix `CTP`